### PR TITLE
Force SSL

### PR DESCRIPTION
**Pull request by grenlayk**
Added `web.xml` file to forse https

### CODE REVIEW

**Review by polinasand — APPROVED** [review]

**Review by zajonc-gugle — APPROVED** [review]
I've never done this, so let's wait for Bartek's approval.

**Review by bprusak — APPROVED** [review]
As far as I know, that's the way it should be configured, but outside of reading about it I have never tried it either. Let's try now.

**Pull request merged**
